src/category_theory/structured_arrow.lean

```diff
@@ -64,6 +64,14 @@ def hom_mk {f f' : structured_arrow S T} (g : f.right ⟶ f'.right) (w : f.hom 
   right := g,
   w' := by { dsimp, simpa using w.symm, }, }
 
+/--
+Given a structured arrow `X ⟶ F(U)`, and an arrow `U ⟶ Y`, we can construct a morphism of
+structured arrow given by `(X ⟶ F(U)) ⟶ (X ⟶ F(U) ⟶ F(Y))`.
+-/
+def hom_mk' {F : C ⥤ D} {X : D} {Y : C}
```

Comment:
Do we really need this definition? I think it looks a little out of place on its own...

Reply:
I tried not to, but I couldn't find a better way to construct those kind of morphisms without using `have` everywhere.